src/widget/wtracktableview.cpp

```diff
@@ -93,6 +95,9 @@ WTrackTableView::WTrackTableView(QWidget * parent,
     m_pBPMMenu = new QMenu(this);
     m_pBPMMenu->setTitle(tr("Change BPM"));
 
+    m_pColorMenu = new QMenu(this);
+    m_pColorMenu->setTitle(tr("Change Color"));
```

Comment:
"Change" or "Choose"?

Reply:
I picked "change" for consistency. But we can also use another verb. How about `Select Color`?
Merriam-Webster says:
- chosen from a number or group by fitness or preference
- to choose (as by fitness or excellence) from a number or group : pick out